code/datums/supplypacks.dm

```diff
@@ -608,7 +608,7 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 					/obj/item/clothing/suit/russofurcoat)
 	cost = 50
 	containertype = /obj/structure/closet/
-	containername = "Russian Clothing"
+	containername = "russian clothing crate"
```

Comment:
Seriously now, make up your minds would ya, I just want to have my slav gear and my beret available, how many more name changes do I need to do.